tests/test_metadata_eq_.py

```diff
@@ -41,34 +37,74 @@ def setUpClass(cls) -> None:
         cls.repo_dir = os.path.join(
             utils.TESTS_DIR, "repository_data", "repository", "metadata"
         )
+        cls.objects = {}
         cls.metadata = {}
         for md in TOP_LEVEL_ROLE_NAMES:
             with open(os.path.join(cls.repo_dir, f"{md}.json"), "rb") as f:
-                cls.metadata[md] = f.read()
+                data = f.read()
+                cls.metadata[md] = data
+                md_name = md[0].upper() + md[1:]
+                cls.objects[md_name] = Metadata.from_bytes(data).signed
+
+        cls.objects["Metadata"] = Metadata(cls.objects["Timestamp"], {})
+        cls.objects["Signed"] = cls.objects["Timestamp"]
+        cls.objects["Key"] = Key(
+            "id", "rsa", "rsassa-pss-sha256", {"public": "foo"}
+        )
+        cls.objects["Role"] = Role(["keyid1", "keyid2"], 3)
+        cls.objects["MetaFile"] = MetaFile(1, 12, {"sha256": "abc"})
+        cls.objects["DelegatedRole"] = DelegatedRole("a", [], 1, False, ["d"])
+        cls.objects["Delegations"] = Delegations(
+            {"keyid": cls.objects["Key"]}, {"a": cls.objects["DelegatedRole"]}
+        )
+        cls.objects["TargetFile"] = TargetFile(
+            1, {"sha256": "abc"}, "file1.txt"
+        )
+
+    # Keys are class names.
+    # Values are a dictionary. In those dictionaries their keys are class
+    # attributes and values are the new attribute worth.
```

Comment:
```suggestion
    # attributes and values are the new attribute value.
```
?

Also, I think the second part of this comment is only useful to someone who already knows what you are using this dictionary for.

Reply:
I will change it with the hope I am clearer.